benchmark-base/src/main/java/org/conscrypt/ServerSocketBenchmark.java

```diff
@@ -52,7 +52,7 @@
     private static final AtomicLong bytesCounter = new AtomicLong();
     private AtomicBoolean recording = new AtomicBoolean();
 
-    ServerSocketBenchmark(Config config) throws Exception {
+    ServerSocketBenchmark(final Config config) throws Exception {
```

Comment:
Android's toolchain doesn't support Java 8's "effectively final" determinations for local variables, so we have to explicitly declare it as final so it can be used inside the closure below.